docs/docs/tutorials/get-started/customize-sign-in-experience.mdx

```diff
@@ -43,13 +44,79 @@ Toggle on "Enable terms of use" and enter the URL to enforce this process. Logto
+| Keys                                                                                        | It refers to the identifier of each locale value.                                                                                        |
+| Language tag                                                                                | eg. en-US                                                                                                                                |
+| Language code                                                                               | eg. en                                                                                                                                   |
+| Country code                                                                                | eg. US                                                                                                                                   |
+| Language name                                                                               | 简体中文                                                                                                                                 |
+| Added language                                                                              | The language added and customized by users.                                                                                              |
+| Logto provided language                                                                     | Logto official languages and stored in Logto original code base.                                                                         |
+
+#### Set your language mode
 
 <img alt="Sign-in experience - Others" src={sieOthers} style={{ maxWidth: 560 }} />
 
+If you want to configure your languages, go to **Sign in Experience** > **Others** > **Language**.
+
+There are two items: **Enable auto-detect** and select a **Default language**.  
+By default the auto-detect mode is on, which means your app detects the user's locale settings and auto switch to the available languages in Logto.  
+The default language in this case is a fallback language if detected language is not supported in Logto.
+
+However, Logto can make it possible by bringing your own translation.  
+Click **Manage language**, select the language tag, add your own translation and save.  
+You will see it is added in the left nav meantime shown in the default language dropdown.
+
+#### Manage your languge
+
+<img alt="Manage language" src={manageLanguage} style={{ maxWidth: 560 }} />
+
+Clicking the manage language button will open a modal.
+
+On the left side they are the languages Logto provided and you can see a tag Logto provided next to the language tag.  
+You can't delete those pre-defined languages but can customize values of each individual key.  
+Your custom values will overwrite Logto source values when it loads in front of the end users.
```

Comment:
nit: should we clearly split it into two different sections? Like, create your own language phrases and modify an existing language phrase.

Reply:
Yeah, in the new version we separated out.